### ISSUE TITLE
ace-jump-mode looks terrible

### ISSUE THREAD

**Issue by velkyel** [john2x/flatui-theme.el#6]
![screen shot 2015-05-29 at 17 39 37](https://cloud.githubusercontent.com/assets/5645472/7886391/139586fa-062a-11e5-90b4-75a951d26d33.png)

**Comment by velkyel** [comment]
my fix:

```lisp
  (progn
    (load-theme 'flatui t)
    (flatui/with-color-variables
      (custom-theme-set-faces
       'flatui
       `(ace-jump-face-background
         ((t (:foreground ,concrete :background ,clouds :inverse-video nil))))
       `(ace-jump-face-foreground
         ((t (:foreground ,alizarin :background ,clouds :inverse-video nil))))))))
```

**Comment by john2x** [comment]
Thanks! I don't use ace-jump-mode, so I just put random values there. Fixed in 1dc6ae1.
